openapi/paths/performance-prediction.yaml

```diff
@@ -0,0 +1,70 @@
+get:
+  summary: Get performance predictions for a configuration
+  description: >
+    This endpoint returns predicted runtime and memory usage
+    for a given configuration of model, optimizer, batch size, and sequence length.
+  operationId: get-performance-prediction
+  tags: [performance-prediction]
+  parameters:
+    - name: model_name
+      in: query
+      required: true
+      schema:
+        type: string
+        enum:
```

Comment:
please change those enum values also to uppercase so that they map with the same enum in `schemas/performance-prediction.yaml`